frameworks/Java/proteus/src/main/java/io/sinistral/models/Fortune.java

```diff
@@ -31,42 +31,17 @@ public static Fortune generate()
 	}
```

Comment:
Everything about `TMP_MESSAGES` and `generate()` leave a bad taste in my mouth. I did not find any occurrences of either being used, but it should probably be removed.